src/styles.css/blocks/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: no-repeat center url(../../images/star-active.svg);
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 > .stars__star:nth-child(n + 5) {
+  background-image: url(../../images/star.svg);
+}
```

Comment:
you should reuse all styles from prev task
I want to see correct counts of active stars when I change class